src/main/java/com/cst438/controller/UserController.java

```diff
@@ -56,8 +60,18 @@ public UserDTO createUser(@RequestBody UserDTO userDTO) {
             // invalid type
             throw  new ResponseStatusException( HttpStatus.BAD_REQUEST, "invalid user type");
         }
+
         userRepository.save(user);
-        return new UserDTO(user.getId(), user.getName(), user.getEmail(), user.getType());
+
+        UserDTO newUser = new UserDTO(
+                user.getId(),
+                user.getName(),
+                user.getEmail(),
+                user.getType()
+        );
+        //FIXME I am doing something wrong here.  Getting error from Gradebook that password cannot be null.
```

Comment:
You can remove this comment if you would like. I made a temp change to Gradebook branch that fixes this issue. Also, I added a FIXME comment to come back to in that branch.